submissions/src/main/kotlin/gov/cdc/prime/reportstream/submissions/config/SecurityConfig.kt

```diff
@@ -0,0 +1,31 @@
+package gov.cdc.prime.reportstream.submissions.config
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.web.SecurityFilterChain
+
+/**
+ * Allow all requests sans any authn/authz checks.
+ */
+@Configuration
+@EnableWebSecurity
```

Comment:
the submissions app is a standard servlet application. I specifically picked reactive for the auth app since I didn't want it to fall over given a burst of requests since the included http client is blocking.

Reply:
ok makes sense. if we're going to mix the two we definitely need to document what's up and why so later we don't copy-pasta ourselves into trouble